config.gradle

```diff
@@ -137,6 +143,8 @@ if (!project.hasProperty('onlyAthena')) {
                 architecture = 'x86'
                 operatingSystem = 'linux'
                 compilerArgs = linuxCompilerArgs
+                CCompilerArgs << linux32BitArg
+                CCompilerArgs = linuxCCompilerArgs
                 compilerArgs << linux32BitArg
```

Comment:
This is inconsistent ordering compared to the other groups (which put this line immediately after the `compilerArgs` line).